boards/shields/m5stack_core2_ext/doc/index.rst

```diff
@@ -0,0 +1,53 @@
+.. _m5stack_core2_ext:
+
+M5Stack-Core2 base shield
+####################################
+
+Overview
+********
+
+M5Stack-Core2 comes with a base shield that is connected to the M5Stack
+extension connector. It features a MPU6886 6-axis motion tracker and a SPM1423
+microphone.
+
+.. figure:: img/m5stack_core2_ext.webp
+        :align: center
+        :alt: M5Stack-Core2-EXT
+        :width: 400 px
+
+        M5Stack-Core2-Extension module
+
+Pins Assignments
+================
+
++----------------------+--------------+
+| Shield Connector Pin | Function     |
++======================+==============+
+| 0                    | GND          |
++----------------------+--------------+
+| 11                   | 3.3V         |
++----------------------+--------------+
+| 16                   | I2C - intSDA |
++----------------------+--------------+
+| 17                   | I2C - intSCL |
++----------------------+--------------+
+
+Programming
+***********
+
+Set ``-DSHIELD=m5stack_core2_ext`` when you invoke ``west build``.
+For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/mpu6050
+   :board: m5stack_core2
+   :shield: m5stack_core2_ext
+   :goals: build
+
+References
+**********
+
+.. target-notes::
+
+.. _datasheet:
+   https://docs.m5stack.com/en/core/core2
```

Comment:
drop unless you actually have links/footnotes.

Reply:
fixed